flexible_event_config.md

```diff
@@ -47,7 +50,165 @@ In general, the approach here is to more flexibly encode the output of the API f
+
+
+### Trigger-data modulus matching example
+
+Given a source with the following registration:
+
+```jsonc
+{
+  "trigger_data_matching": "modulus",
+
+  "trigger_data": [0, 1, 2, 3, 4 5],
+
+  ...
+}
+```
+
+The trigger-data cardinality is 6, so all triggers' `trigger_data` will be taken
+modulus 6 before determining the matching source:
+
+- `{"trigger_data": "0"}` will match the source because `0 % 6 = 0`
+- `{"trigger_data": "1"}` will match the source because `1 % 6 = 1`
+- `{"trigger_data": "2"}` will the source because `2 % 6 = 2`
+- `{"trigger_data": "6"}` will match Spec A because `6 % 6 = 0`
+- `{"trigger_data": "10"}` will match Spec C because `10 % 6 = 4`
+- `{"trigger_data": "11"}` will match Spec A because `11 % 6 = 5`
+
+## Configurations that are equivalent to the current version
+
+The following are equivalent configurations for the API's current event and navigation sources, respectively. Especially for navigation sources, this illustrates why the noise levels are so high relative to event sources to maintain the same epsilon values: navigation sources have a much larger output space.
+
+It is possible that there are multiple configurations that are equivalent, given that some parameters can be set as default or omitted.
+
+### Equivalent event sources
+
+```jsonc
+// Note: most of the fields here are not required to be explicitly listed.
+// Here we list them explicitly just for clarity.
+{
+  "trigger_data_matching": "modulus",
+  "trigger_data": [0, 1],
+  "event_report_windows": {
+    "end_times": [<30 days>]
+  },
+  "max_event_level_reports": 1,
+  ...
+  "expiry": <30 days> // expiry must be greater than or equal to the last element of the end_times
+}
+```
+
+### Equivalent navigation sources
+
+```jsonc
+// Note: most of the fields here are not required to be explicitly listed.
+// Here we list them explicitly just for clarity.
+{
+  "trigger_data_matching": "modulus",
+  "trigger_data": [0, 1, 2, 3, 4, 5, 6, 7],
+  "event_report_windows": {
+    "end_times": [<2 days>, <7 days>, <30 days>]
+  },
+  "max_event_level_reports": 3,
+  ...
+  "expiry": <30 days> // expiry must be greater than or equal to the last element of the end_times
+}
+```
+
+## Additional Examples
+
+### Binary with more frequent reporting
+
+This example configuration supports a developer who wants to learn whether at least one conversion occurred in the first 10 days (regardless of value), but wants to receive reports at more frequent intervals than the default. Again, in this example any trigger that sets `trigger_data` to a value other than 0 is ineligible for attribution. This is why we refer to this use case as _binary_.
+
+```jsonc
+{
+  "max_event_level_reports": 1,
+  "trigger_data_matching": "exact",
+  "trigger_data": [0],
+  "event_report_windows": {
+    // 1 day, 2 days, 3 days, 5 days, 7 days, 10 days represented in seconds
+    "end_times": [86400, 172800, 259200, 432000, 604800, 864000]
+  },
+}
+```
+
+## Attribution Rate Limit
```

Comment:
This is related to the future work for summary buckets, not anything that has already been specified and implemented.

Reply:
Thanks for flagging. I've moved this section to the "ideas for future iteration" section